source/reference/api/reverse.txt

```diff
@@ -0,0 +1,79 @@
+.. _c2c-api-reverse:
+
+===========
+``reverse``
+===========
+
+.. include:: /includes/preview-warning
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Description
+-----------
+
+Reverses the direction of a committed sync operation.
+
+For example, if you have committed a sync operation where ``cluster0``
+is the source and ``cluster1`` is the destination, if you use the
+``reverse`` endpoint, {+c2c-product-name+} will sync onging writes from
+``cluster1`` to ``cluster0``.
+
+Requirements
+------------
+
+To use the ``reverse`` endpoint:
+
+- Your original sync operation must have been started with the
+  ``reversible`` and ``enableUserWriteBlocking`` options set to
+  ``true``.
+
+- {+c2c-product-name+} must be in the ``COMMITTED`` state.
+
+Request
+-------
+
+.. code-block:: http
+   :copyable: false
+
+   POST /api/v1/reverse 
+
+Request Body Parameters
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: includes/api/facts/no-body-parameters.rst
+
+Response
+--------
+
+.. include:: /includes/api/tables/basic-response.rst
+
+Example
+-------
+
+The following reverses the direction of a committed sync operation.
+
+Request
+~~~~~~~
+
+.. literalinclude:: /includes/api/requests/reverse.sh
+   :language: shell
+
+Response
+~~~~~~~~
+
+.. literalinclude:: /includes/api/responses/success.json
+   :language: json
+   :copyable: false
+
+Behavior
+--------
+
+If the ``reverse`` request is successful, {+c2c-product-name+} enters the
+``RUNNING`` state, with replication going in the reverse direction from
+the original sync job.
```

Comment:
We should add another behavior note/link that you can see via the /progress endpoint the mapping direction for the synchronization of the source and destination clusters.